Hackney.Shared.HousingSearch/Domain/Asset/AssetAddress.cs

```diff
@@ -0,0 +1,37 @@
+namespace Hackney.Shared.HousingSearch.Domain.Asset
+{
+    public class AssetAddress
+    {
+        public static AssetAddress Create(string uprn, string addressLine1, string addressLine2,
+            string addressLine3, string addressLine4, string postCode, string postPreamble)
+        {
+            return new AssetAddress(uprn, addressLine1, addressLine2, addressLine3, addressLine4,
+                postCode, postPreamble);
+        }
+
+        public AssetAddress()
```

Comment:
Do we need this here?

Reply:
Not sure, it's in the [API](https://github.com/LBHackney-IT/housing-search-api/blob/master/HousingSearchApi/V1/Domain/Asset/AssetAddress.cs). @evak6979?